Source/Core/InputCommon/ControllerEmu/ControlGroup/Cursor.cpp

```diff
@@ -30,13 +32,25 @@ Cursor::Cursor(const std::string& name_) : ControlGroup(name_, GroupType::Cursor
   controls.emplace_back(std::make_unique<Input>(Translate, _trans("Backward")));
   controls.emplace_back(std::make_unique<Input>(Translate, _trans("Hide")));
   controls.emplace_back(std::make_unique<Input>(Translate, _trans("Recenter")));
+  if(Libretro::Options::irMode==1){
+    // Set right stick to control the IR
+    numeric_settings.emplace_back(std::make_unique<NumericSetting>(_trans("Center"), 0.5));
+    numeric_settings.emplace_back(std::make_unique<NumericSetting>(_trans("Width"), 0.1));
+    numeric_settings.emplace_back(std::make_unique<NumericSetting>(_trans("Height"), 0.2));
+    numeric_settings.emplace_back(std::make_unique<NumericSetting>(_trans("Dead Zone"), 0, 0, 4));
+    boolean_settings.emplace_back(std::make_unique<BooleanSetting>(_trans("Relative Input"), true));
+    boolean_settings.emplace_back(std::make_unique<BooleanSetting>(_trans("Auto-Hide"), true));
+  }
+  else {
+    // Mouse controls IR
+    numeric_settings.emplace_back(std::make_unique<NumericSetting>(_trans("Center"), 0.5));
+    numeric_settings.emplace_back(std::make_unique<NumericSetting>(_trans("Width"), 0.5));
+    numeric_settings.emplace_back(std::make_unique<NumericSetting>(_trans("Height"), 0.5));
+    numeric_settings.emplace_back(std::make_unique<NumericSetting>(_trans("Dead Zone"), 0, 0, 20));
+    boolean_settings.emplace_back(std::make_unique<BooleanSetting>(_trans("Relative Input"), true));
```

Comment:
Relative input was false for screen pointer/mouse.
Auto-hide was false.
Any reason to change these values?

Reply:
Nope, just  copy and paste error here.